src/components/home/latestItems.js

```diff
@@ -1,18 +1,60 @@
-import React from 'react';
+//  import modules
+import React, { Component, PropTypes } from 'react';
+import { bindActionCreators } from 'redux';
+import { connect } from 'react-redux';
+//  import actions
+import { homeActions } from '../../redux/actions/index';
+//  import components
 import { SeparateLine } from '../separateLine';
 import { LatestView } from './latestView';
+// import styles
 import styles from '../../../static/css/app.css';
 
+const fetchLatestItemsData = props => props.fetchLatestAction();
+const getItemKey = (name, uid) => name + uid;
 
-export const LatestItems = () => (
-  <article className={styles.latesList}>
-    <header>
-      <SeparateLine text="Latest" />
-    </header>
-    <content>
-      {
-        [1, 2, 3].map((item, index) => <LatestView number={index + 1} key={index} />)
-      }
-    </content>
-  </article>
-);
+class LatestItems extends Component {
+  constructor(props) {
+    super(props);
+    fetchLatestItemsData(this.props);
+  }
+
+  getItemView() {
+    if (this.props.items !== []) {
+      return this.props.items.map(
+        (item, index) => <LatestView
+          {...item}
+          number={index + 1}
+          key={getItemKey('latest_', item.id)}
```

Comment:
What's the reason to use one more function if you can use es6 template string `latest_${item.id}`